.github/workflows/e2e-nightly-37x.yml

```diff
@@ -0,0 +1,104 @@
+# Runs randomly generated E2E testnets nightly on the v0.37.x branch.
```

Comment:
Assuming you took `.github/workflows/e2e-nightly-34x.yml` and mechanically adapted it.
Let me know if you did otherwise.

Reply:
It's actually a combination of `.github/workflows/e2e-nightly-{main,34x}.yml`. The `main` E2E tests run more testnet groups.

Reply:
OK, checking out your branch and melding the files.